Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -39,11 +40,14 @@ import ReportFilter, { ReportFilterBuilder } from "@/models/reportFilter";
 export default class ReportsView extends Vue {
     @Getter("webClient", { namespace: "config" })
     config!: WebClientConfiguration;
+
     @Getter("patientData", { namespace: "user" })
     patientData!: PatientData;
     @Getter("medicationStatements", { namespace: "medication" })
     medicationStatements!: MedicationStatementHistory[];
 
+    @Getter("user", { namespace: "user" }) user!: User;
```

Comment:
is this necessary? I dont see it being used

Reply:
it passes the hdid to the getPatientData call

Reply:
The action already has access to the context so is not necessary

Reply:
okay it just follows the same behaviour for all the calls in the user store, (e.g. checkRegistration)
Since i have to change this i'm removing the hdid parameter from all the action methods